package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "instantsearch.js",
-  "version": "2.2.0",
+  "version": "2.2.1-beta.0",
```

Comment:
this should only change with the build script

Reply:
That's alright, since it's gonna be 2.2.1 anyway.